python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -413,6 +413,28 @@ def ElementwiseLeakyReluModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ElementwisePreluModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+        ([-1], torch.float32, True),
+    ])
+    def forward(self, x, weight):
+        return torch.ops.aten.prelu(x, weight)
+
+@register_test_case(module_factory=lambda: ElementwisePreluModule())
+def ElementwisePreluModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(1, 1, 1), tu.rand(1) )
```

Comment:
Can you make the input tensors a bit larger to make the test more comprehensive? Also, the input tensors should have both positive and negative numbers to test the negative and positive logic of the lowering